index.html

```diff
@@ -0,0 +1,212 @@
+  <link rel="connect" href="InaiMathi.ttf">
+  <link rel="preconnect" href="https://fonts.gstatic.com"> 
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+  <link
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+    type="text/css"
+  />
+  <link href="styles.css" rel="stylesheet" />
+</head>
+<body>
+  <header>
+    <nav class="header">
+      <a href="#" class="log">Kaneez Fatima</a>
+      <a><i class="fa fa-bars cursor"></i></a>
+    </nav>
+  </header>
+      <section class="main-section img">
+        <div>
+          <h1>Hey There.</h1>
+            <h1>I'm Kaneez Fatima</h1>
+            <h2>I'am a Software Developer</h2>
+          
+    
+          <p class="para">
+            I can help you build a product, feature or website. Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don't hestiate to contact me.
+          </p>
+    
+          <ul>
+         <li class="inline"><a
+            href="https://github.com/Kaneez98Fatima"
+            target="_blank" rel="noopener noreferrer"><i class="fa fa-github icon"></i></a></li>
+          <li class="inline"><a
+            href="https://www.linkedin.com/in/kaneez-fatima-0a86601b3"
+            target="_blank"
+            rel="noopener noreferrer "
+            ><i class="fa fa-linkedin icon"></i
+          ></a></li>
+         <li class="inline"> <a
+            href="https://www.instagram.com/fatima98kaneez."
+            target="_blank"
+            rel="noopener noreferrer"
+            ><i class="fa fa-instagram icon"></i></a></li>
+             <li class="inline"><a href="Mail:Kaneez98fatima@gmail.com" target="_blank" rel="noopener"
+                ><i class="fa fa-envelope icon"></i
+              ></a></li>
+            </ul>
+            </div>
+        </section>
+        <section class="work-area">
+          <h4 class="recent-work">My Recent Works </h4>
+          <div class="works">
+            <div class="work-img"></div>
+            <div class="cap">
+              <h3 class="H3-W">Multi-Post Stories</h3>
+              <p class="para-1">
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text ever
+                  since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+              </div>
+              <div class="hor">
+                <a href="#" class="button">css</a>
+                <button type="button" class="button">Html</button>
+                <button type="button" class="button">Bootstarp</button>
+                <button type="button" class="button">Ruby</button>
+                </div>
```

Comment:
Kindly compare this section's design with the original design and make necessary amendments 
![Screenshot (60)](https://user-images.githubusercontent.com/52497246/121771516-af6a9580-cb67-11eb-8125-4d6245e9abcd.png)
![Screenshot (61)](https://user-images.githubusercontent.com/52497246/121771520-b2fe1c80-cb67-11eb-80ca-a340a8b3ba5b.png)

Figma Design

![Screenshot (63)](https://user-images.githubusercontent.com/52497246/121771574-0a9c8800-cb68-11eb-9399-5e70b138a28d.png)
![Screenshot (62)](https://user-images.githubusercontent.com/52497246/121771581-15efb380-cb68-11eb-88c2-6461f0d6b1aa.png)